app/controllers/holdings_controller.rb

```diff
@@ -0,0 +1,103 @@
+class HoldingsController < ApplicationController
+    before_action :authenticate_user!
+
+    def new
+        @stock = Stock.find(params[:stock_id])
+        @holding = current_user.holdings.build
+        @wallet = current_user.wallet
+        @buy_value = helpers.stock_total_value(true, @stock.current_price)
+        @sell_value = helpers.stock_total_value(false, @stock.current_price)
+        @current_stock = current_user.holdings.find_by(ticker: @stock.ticker)
+
```

Comment:
These are for the variables in the form at buy and selling stocks. So we can access the databases.
![image](https://user-images.githubusercontent.com/86232999/148640013-b28f825e-8d02-4786-9151-8338c35f736e.png)

helpers.stock_total_value => a function at application module
![image](https://user-images.githubusercontent.com/86232999/148641051-369ba9f3-7a9d-4409-b3d8-aad4c5046b7e.png)